env.prod.example

```diff
@@ -9,4 +9,9 @@ DB_PORT="<SET_THIS>"
 DB_SSL_MODE="<SET_THIS>"
 DB_PASSWORD="<SET_THIS>"
 SITE_URL="<SET_THIS>"
-DJANGO_SETTINGS_MODULE="response.settings.prod"
\ No newline at end of file
+DJANGO_SETTINGS_MODULE="response.settings.prod"
+ENCRYPTED_FIELD_KEY="<SET_THIS>"
+PAGERDUTY_ENABLED="<True|False>"
+PAGERDUTY_API_KEY="<SET_THIS_IF_PAGERDUTY_IS_ENABLED>"
+PAGERDUTY_SERVICE="<SET_THIS_IF_PAGERDUTY_IS_ENABLED>"
+PAGERDUTY_DEFAULT_EMAIL="<SET_THIS_IF_PAGERDUTY_IS_ENABLED>"
```

Comment:
Can you add a newline to make github happy?

Reply:
@adamlphillips Added.